site/tests/conftest.py

```diff
@@ -84,3 +103,214 @@ def client_with_login(client, users):
     login_user(user)
     login_user_via_session(client, email=user.email)
     return client
+
+
+@pytest.fixture(scope="module")
+def resource_type_v(app):
```

Comment:
most of these are copy pasted from rdm-records/app-rdm records. It makes me wonder if we should have sort of an RDM specific subset of pytets fixtures. cc @zzacharo @kpsherva, it might be worth discussing this at some point.